src/geocel/vg/VecgeomParams.cc

```diff
@@ -531,11 +532,7 @@ void VecgeomParams::build_volume_tracking()
 
     {
         ScopedTimeAndRedirect time_and_output_("vecgeom::ABBoxManager");
-#if VECGEOM_VERSION < 0x020000
-        vecgeom::ABBoxManager::Instance().InitABBoxesForCompleteGeometry();
-#else
-        vecgeom::ABBoxManager_t::Instance().InitABBoxesForCompleteGeometry();
-#endif
+        detail::ABBoxManager_t::Instance().InitABBoxesForCompleteGeometry();
     }
```

Comment:
Are we still keeping the compatibility with VecGeom 1.2.10 after this PR?

Reply:
Yep!